torch/testing/_internal/common_pruning.py

```diff
@@ -8,7 +8,16 @@
 
 
 def elements_are_subset(subset_tensor, superset_tensor):
-    return all(i in superset_tensor for i in subset_tensor)
+    i = 0
+    for row in subset_tensor:
+        while i < len(superset_tensor):
+            if not torch.equal(row, superset_tensor[i]):
+                i += 1
+            else:
+                break
+        else:
+            return False
+    return True
```

Comment:
Some comments regarding assumptions (i.e. subset and order preserved) would help future use of this function, since it no longer matches the function name.